src/components/Status/__stories__/Animated.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import { Template } from './Template.stories'
+
+export const Animated = Template.bind({})
+Animated.args = {
+  animated: true,
+  variant: 'info',
+}
+Animated.parameters = {
+  docs: {
+    description: {
+      story:
+        'Animated prop will make StatusIndicator blink. This prop is usually used for temporary or "in progress" state.',
+    },
+  },
+}
```

Comment:
Better to let it breath a bit don't you think?

```suggestion
export const Animated = Template.bind({})

Animated.args = {
  animated: true,
  variant: 'info',
}

Animated.parameters = {
  docs: {
    description: {
      story:
        'Animated prop will make StatusIndicator blink. This prop is usually used for temporary or "in progress" state.',
    },
  },
}
```